addon/templates/components/frost-list-item-content.hbs

```diff
@@ -0,0 +1,67 @@
+<div class={{concat
+    'frost-list-item-container'
+    (if (eq index 0) ' first')
+    (if (eq index (sub _items.length 1)) ' last')
+    (if model.states.isSelected ' is-selected')
+  }}
+  data-test={{hook (concat hook '-item-container') index=index }}
+>
+  <div class='frost-list-item-container-base'>
+    {{#if (and (or itemExpansion isTypedItemWithExpansion) (not isTypedItemWithoutExpansion))}}
```

Comment:
Let's manage all this the same way and avoid this `and`

Reply:
I have moved this logic into a CP named `isItemExpansionVisible`.